### PR TITLE
Change kuttl job to use 'base-simple-crc' as parent job

### PR DESCRIPTION
**Pull request by viroel**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by viroel** [comment]
recheck

issue with the inventory file?

**Comment by viroel** [comment]
recheck

File: zuul.d/kuttl.yaml [hunk 1]
```diff
@@ -23,9 +27,11 @@
       - ^zuul.d/kuttl.yaml
     vars:
       cifmw_kuttl_tests_operator_list:
-        - cinder
+        - keystone
```
Comment (viroel) [inline zuul.d/kuttl.yaml:30]: You may ask why, but Cinder kuttl tests aren't so stable atm: https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl

**Review by raukadah — APPROVED** [review]
/lgtm

**Pull request merged**